test/routes/objects.test.js

```diff
@@ -536,7 +575,154 @@ test('Objects API', async (t) => {
       env: 'ote'
     });
 
-    t.deepEqual(head101, { headVersion: '1.0.1', latestVersion: '2.0.0' });
+    t.same(head101, { headVersion: '1.0.1', latestVersion: '2.0.0' });
   });
+
+  t.test('create object in all predefinied enviroments', async (t) => {
+    t.plan(3);
+
+    await Promise.all([
+      createEnv(fastify, {
+        name: 'preObj0',
+        env: 'devo'
+      }),
+      createEnv(fastify, {
+        name: 'preObj1',
+        env: 'test'
+      }),
+      createEnv(fastify, {
+        name: 'preObj1',
+        env: 'prod'
+      })
+    ]);
+
+    const res = await fastify.inject({
+      method: 'POST',
+      url: '/objects',
+      headers: {
+        'Content-type': 'application/json'
+      },
+      payload: JSON.stringify({
+        name: 'preObj1',
+        version: '1.0.0',
+        data: 'data from CDN api',
+        variant: 'en-US'
+      })
+    });
+
+    t.equal(res.statusCode, 201);
+
+    const body = JSON.parse(res.payload);
+    t.same(body, { created: true });
+
+    const head = await getHeads(fastify, {
+      name: 'preObj1'
+    });
+    t.same(head, [
+      {
+        enviroment: 'prod',
```

Comment:
Basically find/replace all "enviroment" with "environment".

```suggestion
        environment: 'prod',
```